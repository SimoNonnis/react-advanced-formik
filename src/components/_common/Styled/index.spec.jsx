/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { theme } from '@housesimple/react-components';

import {
  FieldRow,
  FieldBox,
  Fieldset,
  FieldsetRadios,
  LegendWrapper,
  Legend,
  LegendRadios,
  ErrorRadios,
  ControlButton,
  ContainerProofOf,
  LegendProofOf,
  AcceptedDocument,
  ConfirmationIcon,
  DocumentName,
  DeleteDocument,
  Remove,
  OpenGalleryButton,
  IconUpload,
  SubmitButtonWrapper,
  LoaderWrapper,
  AdditionalInfo,
} from './';

describe('Test _common styled components', () => {
  describe('Test <FieldRow /> component', () => {
    it('should render FieldRow correctly with theme', () => {
      const wrapper = shallow(<FieldRow theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldRow correctly passing alignBottom', () => {
      const wrapper = shallow(<FieldRow theme={theme} alignBottom />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldRow correctly passing noSpaceBetween', () => {
      const wrapper = shallow(<FieldRow theme={theme} noSpaceBetween />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <FieldBox /> component', () => {
    it('should render FieldBox correctly with theme', () => {
      const wrapper = shallow(<FieldBox theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing center', () => {
      const wrapper = shallow(<FieldBox theme={theme} center />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing small', () => {
      const wrapper = shallow(<FieldBox theme={theme} small />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing medium', () => {
      const wrapper = shallow(<FieldBox theme={theme} medium />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing large', () => {
      const wrapper = shallow(<FieldBox theme={theme} large />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing xlarge', () => {
      const wrapper = shallow(<FieldBox theme={theme} xlarge />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing fullWidth', () => {
      const wrapper = shallow(<FieldBox theme={theme} fullWidth />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render FieldBox correctly passing marginLeft', () => {
      const wrapper = shallow(<FieldBox theme={theme} marginLeft />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <Fieldset /> component', () => {
    it('should render Fieldset correctly with theme', () => {
      const wrapper = shallow(<Fieldset theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <FieldsetRadios /> component', () => {
    it('should render FieldsetRadios correctly with theme', () => {
      const wrapper = shallow(<FieldsetRadios theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <LegendWrapper /> component', () => {
    it('should render LegendWrapper correctly with theme', () => {
      const wrapper = shallow(<LegendWrapper theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <Legend /> component', () => {
    it('should render Legend correctly with theme', () => {
      const wrapper = shallow(<Legend theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <LegendRadios /> component', () => {
    it('should render LegendRadios correctly with theme', () => {
      const wrapper = shallow(<LegendRadios theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render LegendRadios correctly with error', () => {
      const wrapper = shallow(<LegendRadios theme={theme} error />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <ErrorRadios /> component', () => {
    it('should render ErrorRadios correctly with theme', () => {
      const wrapper = shallow(<ErrorRadios theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <ContainerProofOf /> component', () => {
    it('should render ContainerProofOf correctly with theme', () => {
      const wrapper = shallow(<ContainerProofOf theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <LegendProofOf /> component', () => {
    it('should render LegendProofOf correctly with theme', () => {
      const wrapper = shallow(<LegendProofOf theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <AcceptedDocument /> component', () => {
    it('should render AcceptedDocument correctly with theme', () => {
      const wrapper = shallow(<AcceptedDocument theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <ConfirmationIcon /> component', () => {
    it('should render ConfirmationIcon correctly with theme', () => {
      const wrapper = shallow(<ConfirmationIcon theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <DocumentName /> component', () => {
    it('should render DocumentName correctly with theme', () => {
      const wrapper = shallow(<DocumentName theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <DeleteDocument /> component', () => {
    it('should render DeleteDocument correctly with theme', () => {
      const wrapper = shallow(
        <DeleteDocument onClick={() => {}} theme={theme}>
          Delete
        </DeleteDocument>
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <Remove /> component', () => {
    it('should render Remove correctly with theme', () => {
      const wrapper = shallow(<Remove theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <OpenGalleryButton /> component', () => {
    it('should render OpenGalleryButton correctly with theme', () => {
      const wrapper = shallow(
        <OpenGalleryButton onClick={() => {}} theme={theme}>
          Open
        </OpenGalleryButton>
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <IconUpload /> component', () => {
    it('should render IconUpload correctly with theme', () => {
      const wrapper = shallow(<IconUpload theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <ControlButton /> component', () => {
    it('should render ControlButton correctly with theme', () => {
      const wrapper = shallow(
        <ControlButton onClick={() => {}} theme={theme}>
          Remove
        </ControlButton>
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <SubmitButtonWrapper /> component', () => {
    it('should render SubmitButtonWrapper correctly with theme', () => {
      const wrapper = shallow(<SubmitButtonWrapper theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render SubmitButtonWrapper with noPad', () => {
      const wrapper = shallow(<SubmitButtonWrapper noPad theme={theme} />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
  describe('Test <LoaderWrapper /> component', () => {
    it('should render LoaderWrapper correctly with theme', () => {
      const wrapper = shallow(<LoaderWrapper />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });

  describe('Test <AdditionalInfo /> component', () => {
    it('should render AdditionalInfo correctly', () => {
      const wrapper = shallow(<AdditionalInfo />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
