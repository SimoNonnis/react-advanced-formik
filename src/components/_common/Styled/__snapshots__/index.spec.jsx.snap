// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test _common styled components Test <AcceptedDocument /> component should render AcceptedDocument correctly with theme 1`] = `
<div
  className="sc-VigVT dfOJkv"
/>
`;

exports[`Test _common styled components Test <AdditionalInfo /> component should render AdditionalInfo correctly 1`] = `
<span
  className="sc-kpOJdX kFnvZr"
/>
`;

exports[`Test _common styled components Test <ConfirmationIcon /> component should render ConfirmationIcon correctly with theme 1`] = `
<Confirmation
  className="sc-jTzLTM frXRWe"
  classNamePath="sizeIconPath"
  fill="currentColor"
  height={40}
  theme={
    Object {
      "base": Object {
        "fontFamily": "\\"proxima-nova\\", \\"Open Sans\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
        "fontSize": "14px",
        "fontWeight": 400,
        "lineHeight": 1.45,
      },
      "breakpoints": Array [
        504,
        671,
        867,
        1050,
        1200,
      ],
      "color": Object {
        "black": "#000000",
        "coral": "#F4364C",
        "darkGrey": "#77838d",
        "green": "#35A856",
        "lightBlue1": "#daf3ff",
        "lightBlue2": "#bfe8f8",
        "lightGreen": "#B4E2C2",
        "lightGrey1": "#d6d6d6",
        "lightGrey2": "#e6e6e6",
        "lightGrey3": "#f6f6f6",
        "lightGrey4": "#c8cccf",
        "lightOrange": "#ffeaca",
        "lightPink": "#f8d7e0",
        "mediumBlue": "#0099cf",
        "mediumGrey": "#979797",
        "orange": "#ffad2f",
        "pink": "#e46185",
        "red": "#f80000",
        "red15": "#fedede",
        "stoneGrey": "#a6adb4",
        "teal": "#06A693",
        "veryDarkBlue": "#1D3141",
        "veryDarkBlue20": "#C8CCCF",
        "veryDarkBlue40": "#9DA5AB",
        "veryDarkBlue60": "#717D87",
        "veryDarkBlue7": "#EEF0F3",
        "veryDarkBlue80": "#4A5A67",
        "white": "#ffffff",
      },
      "maxWidth": "1200px",
      "space": Array [
        5,
        10,
        15,
        20,
        30,
      ],
      "spacing": Object {
        "lg": "28px",
        "md": "18px",
        "sm": "10px",
        "xl": "36px",
        "xs": "5px",
      },
      "typography": Object {
        "caption": Object {
          "desktop": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
          "phone": Object {
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.571rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
        },
        "h1": Object {
          "desktop": Object {
            "fontSize": "2.286rem",
            "lineHeight": "2.571rem",
          },
          "phone": Object {
            "fontSize": "1.714rem",
            "fontWeight": 600,
            "lineHeight": "2rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "2rem",
            "lineHeight": "2.429rem",
          },
        },
        "h2": Object {
          "desktop": Object {
            "fontSize": "1.571rem",
            "lineHeight": "1.857rem",
          },
          "phone": Object {
            "fontSize": "1.429rem",
            "fontWeight": 600,
            "lineHeight": "1.714rem",
            "margin": "0 0 18px 0",
          },
          "tablet": Object {
            "fontSize": "1.429rem",
            "lineHeight": "1.714rem",
          },
        },
        "h3": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 700,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
        "intro": Object {
          "desktop": Object {
            "fontSize": "1.357rem",
            "lineHeight": "1.929rem",
          },
          "phone": Object {
            "fontSize": "1.214rem",
            "fontWeight": 300,
            "lineHeight": "1.714rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "1.214rem",
            "lineHeight": "1.714rem",
          },
        },
        "p": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 400,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
      },
    }
  }
  width={40}
/>
`;

exports[`Test _common styled components Test <ContainerProofOf /> component should render ContainerProofOf correctly with theme 1`] = `
<div
  className="sc-gZMcBi dwygME"
/>
`;

exports[`Test _common styled components Test <ControlButton /> component should render ControlButton correctly with theme 1`] = `
<Button
  className="sc-iwsKbI cGpPKR"
  disabled={false}
  fullWidth={false}
  loading={false}
  onClick={[Function]}
  rounded={3}
  size=""
  styleType=""
  theme={
    Object {
      "base": Object {
        "fontFamily": "\\"proxima-nova\\", \\"Open Sans\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
        "fontSize": "14px",
        "fontWeight": 400,
        "lineHeight": 1.45,
      },
      "breakpoints": Array [
        504,
        671,
        867,
        1050,
        1200,
      ],
      "color": Object {
        "black": "#000000",
        "coral": "#F4364C",
        "darkGrey": "#77838d",
        "green": "#35A856",
        "lightBlue1": "#daf3ff",
        "lightBlue2": "#bfe8f8",
        "lightGreen": "#B4E2C2",
        "lightGrey1": "#d6d6d6",
        "lightGrey2": "#e6e6e6",
        "lightGrey3": "#f6f6f6",
        "lightGrey4": "#c8cccf",
        "lightOrange": "#ffeaca",
        "lightPink": "#f8d7e0",
        "mediumBlue": "#0099cf",
        "mediumGrey": "#979797",
        "orange": "#ffad2f",
        "pink": "#e46185",
        "red": "#f80000",
        "red15": "#fedede",
        "stoneGrey": "#a6adb4",
        "teal": "#06A693",
        "veryDarkBlue": "#1D3141",
        "veryDarkBlue20": "#C8CCCF",
        "veryDarkBlue40": "#9DA5AB",
        "veryDarkBlue60": "#717D87",
        "veryDarkBlue7": "#EEF0F3",
        "veryDarkBlue80": "#4A5A67",
        "white": "#ffffff",
      },
      "maxWidth": "1200px",
      "space": Array [
        5,
        10,
        15,
        20,
        30,
      ],
      "spacing": Object {
        "lg": "28px",
        "md": "18px",
        "sm": "10px",
        "xl": "36px",
        "xs": "5px",
      },
      "typography": Object {
        "caption": Object {
          "desktop": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
          "phone": Object {
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.571rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
        },
        "h1": Object {
          "desktop": Object {
            "fontSize": "2.286rem",
            "lineHeight": "2.571rem",
          },
          "phone": Object {
            "fontSize": "1.714rem",
            "fontWeight": 600,
            "lineHeight": "2rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "2rem",
            "lineHeight": "2.429rem",
          },
        },
        "h2": Object {
          "desktop": Object {
            "fontSize": "1.571rem",
            "lineHeight": "1.857rem",
          },
          "phone": Object {
            "fontSize": "1.429rem",
            "fontWeight": 600,
            "lineHeight": "1.714rem",
            "margin": "0 0 18px 0",
          },
          "tablet": Object {
            "fontSize": "1.429rem",
            "lineHeight": "1.714rem",
          },
        },
        "h3": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 700,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
        "intro": Object {
          "desktop": Object {
            "fontSize": "1.357rem",
            "lineHeight": "1.929rem",
          },
          "phone": Object {
            "fontSize": "1.214rem",
            "fontWeight": 300,
            "lineHeight": "1.714rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "1.214rem",
            "lineHeight": "1.714rem",
          },
        },
        "p": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 400,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
      },
    }
  }
  type="submit"
>
  Remove
</Button>
`;

exports[`Test _common styled components Test <DeleteDocument /> component should render DeleteDocument correctly with theme 1`] = `
<Button
  className="sc-jzJRlG fITnIK"
  disabled={false}
  fullWidth={false}
  loading={false}
  onClick={[Function]}
  rounded={3}
  size=""
  styleType=""
  theme={
    Object {
      "base": Object {
        "fontFamily": "\\"proxima-nova\\", \\"Open Sans\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
        "fontSize": "14px",
        "fontWeight": 400,
        "lineHeight": 1.45,
      },
      "breakpoints": Array [
        504,
        671,
        867,
        1050,
        1200,
      ],
      "color": Object {
        "black": "#000000",
        "coral": "#F4364C",
        "darkGrey": "#77838d",
        "green": "#35A856",
        "lightBlue1": "#daf3ff",
        "lightBlue2": "#bfe8f8",
        "lightGreen": "#B4E2C2",
        "lightGrey1": "#d6d6d6",
        "lightGrey2": "#e6e6e6",
        "lightGrey3": "#f6f6f6",
        "lightGrey4": "#c8cccf",
        "lightOrange": "#ffeaca",
        "lightPink": "#f8d7e0",
        "mediumBlue": "#0099cf",
        "mediumGrey": "#979797",
        "orange": "#ffad2f",
        "pink": "#e46185",
        "red": "#f80000",
        "red15": "#fedede",
        "stoneGrey": "#a6adb4",
        "teal": "#06A693",
        "veryDarkBlue": "#1D3141",
        "veryDarkBlue20": "#C8CCCF",
        "veryDarkBlue40": "#9DA5AB",
        "veryDarkBlue60": "#717D87",
        "veryDarkBlue7": "#EEF0F3",
        "veryDarkBlue80": "#4A5A67",
        "white": "#ffffff",
      },
      "maxWidth": "1200px",
      "space": Array [
        5,
        10,
        15,
        20,
        30,
      ],
      "spacing": Object {
        "lg": "28px",
        "md": "18px",
        "sm": "10px",
        "xl": "36px",
        "xs": "5px",
      },
      "typography": Object {
        "caption": Object {
          "desktop": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
          "phone": Object {
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.571rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
        },
        "h1": Object {
          "desktop": Object {
            "fontSize": "2.286rem",
            "lineHeight": "2.571rem",
          },
          "phone": Object {
            "fontSize": "1.714rem",
            "fontWeight": 600,
            "lineHeight": "2rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "2rem",
            "lineHeight": "2.429rem",
          },
        },
        "h2": Object {
          "desktop": Object {
            "fontSize": "1.571rem",
            "lineHeight": "1.857rem",
          },
          "phone": Object {
            "fontSize": "1.429rem",
            "fontWeight": 600,
            "lineHeight": "1.714rem",
            "margin": "0 0 18px 0",
          },
          "tablet": Object {
            "fontSize": "1.429rem",
            "lineHeight": "1.714rem",
          },
        },
        "h3": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 700,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
        "intro": Object {
          "desktop": Object {
            "fontSize": "1.357rem",
            "lineHeight": "1.929rem",
          },
          "phone": Object {
            "fontSize": "1.214rem",
            "fontWeight": 300,
            "lineHeight": "1.714rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "1.214rem",
            "lineHeight": "1.714rem",
          },
        },
        "p": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 400,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
      },
    }
  }
  type="submit"
>
  Delete
</Button>
`;

exports[`Test _common styled components Test <DocumentName /> component should render DocumentName correctly with theme 1`] = `
<span
  className="sc-fjdhpX cdlzIV"
/>
`;

exports[`Test _common styled components Test <ErrorRadios /> component should render ErrorRadios correctly with theme 1`] = `
<div
  className="sc-dnqmqq drflnR"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing center 1`] = `
<div
  className="sc-bxivhb fxyiUS"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing fullWidth 1`] = `
<div
  className="sc-bxivhb jrstJt"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing large 1`] = `
<div
  className="sc-bxivhb idONSX"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing marginLeft 1`] = `
<div
  className="sc-bxivhb KhTYA"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing medium 1`] = `
<div
  className="sc-bxivhb iwZbsS"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing small 1`] = `
<div
  className="sc-bxivhb kUATqC"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly passing xlarge 1`] = `
<div
  className="sc-bxivhb dRErhJ"
/>
`;

exports[`Test _common styled components Test <FieldBox /> component should render FieldBox correctly with theme 1`] = `
<div
  className="sc-bxivhb iqLnav"
/>
`;

exports[`Test _common styled components Test <FieldRow /> component should render FieldRow correctly passing alignBottom 1`] = `
<div
  className="sc-htpNat IqKBU"
/>
`;

exports[`Test _common styled components Test <FieldRow /> component should render FieldRow correctly passing noSpaceBetween 1`] = `
<div
  className="sc-htpNat feGspz"
/>
`;

exports[`Test _common styled components Test <FieldRow /> component should render FieldRow correctly with theme 1`] = `
<div
  className="sc-htpNat fzmrUj"
/>
`;

exports[`Test _common styled components Test <Fieldset /> component should render Fieldset correctly with theme 1`] = `
<fieldset
  className="sc-ifAKCX ZwIhd"
/>
`;

exports[`Test _common styled components Test <FieldsetRadios /> component should render FieldsetRadios correctly with theme 1`] = `
<fieldset
  className="sc-EHOje kGiAhO"
/>
`;

exports[`Test _common styled components Test <IconUpload /> component should render IconUpload correctly with theme 1`] = `
<UploadIcon
  className="sc-chPdSV kjpoUG"
  fill="currentColor"
  height={40}
  theme={
    Object {
      "base": Object {
        "fontFamily": "\\"proxima-nova\\", \\"Open Sans\\", \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
        "fontSize": "14px",
        "fontWeight": 400,
        "lineHeight": 1.45,
      },
      "breakpoints": Array [
        504,
        671,
        867,
        1050,
        1200,
      ],
      "color": Object {
        "black": "#000000",
        "coral": "#F4364C",
        "darkGrey": "#77838d",
        "green": "#35A856",
        "lightBlue1": "#daf3ff",
        "lightBlue2": "#bfe8f8",
        "lightGreen": "#B4E2C2",
        "lightGrey1": "#d6d6d6",
        "lightGrey2": "#e6e6e6",
        "lightGrey3": "#f6f6f6",
        "lightGrey4": "#c8cccf",
        "lightOrange": "#ffeaca",
        "lightPink": "#f8d7e0",
        "mediumBlue": "#0099cf",
        "mediumGrey": "#979797",
        "orange": "#ffad2f",
        "pink": "#e46185",
        "red": "#f80000",
        "red15": "#fedede",
        "stoneGrey": "#a6adb4",
        "teal": "#06A693",
        "veryDarkBlue": "#1D3141",
        "veryDarkBlue20": "#C8CCCF",
        "veryDarkBlue40": "#9DA5AB",
        "veryDarkBlue60": "#717D87",
        "veryDarkBlue7": "#EEF0F3",
        "veryDarkBlue80": "#4A5A67",
        "white": "#ffffff",
      },
      "maxWidth": "1200px",
      "space": Array [
        5,
        10,
        15,
        20,
        30,
      ],
      "spacing": Object {
        "lg": "28px",
        "md": "18px",
        "sm": "10px",
        "xl": "36px",
        "xs": "5px",
      },
      "typography": Object {
        "caption": Object {
          "desktop": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
          "phone": Object {
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": "1.571rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1rem",
            "lineHeight": "1.571rem",
          },
        },
        "h1": Object {
          "desktop": Object {
            "fontSize": "2.286rem",
            "lineHeight": "2.571rem",
          },
          "phone": Object {
            "fontSize": "1.714rem",
            "fontWeight": 600,
            "lineHeight": "2rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "2rem",
            "lineHeight": "2.429rem",
          },
        },
        "h2": Object {
          "desktop": Object {
            "fontSize": "1.571rem",
            "lineHeight": "1.857rem",
          },
          "phone": Object {
            "fontSize": "1.429rem",
            "fontWeight": 600,
            "lineHeight": "1.714rem",
            "margin": "0 0 18px 0",
          },
          "tablet": Object {
            "fontSize": "1.429rem",
            "lineHeight": "1.714rem",
          },
        },
        "h3": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 700,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
        "intro": Object {
          "desktop": Object {
            "fontSize": "1.357rem",
            "lineHeight": "1.929rem",
          },
          "phone": Object {
            "fontSize": "1.214rem",
            "fontWeight": 300,
            "lineHeight": "1.714rem",
            "margin": "0 0 36px 0",
          },
          "tablet": Object {
            "fontSize": "1.214rem",
            "lineHeight": "1.714rem",
          },
        },
        "p": Object {
          "desktop": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
          "phone": Object {
            "fontSize": "1.143rem",
            "fontWeight": 400,
            "lineHeight": "1.714rem",
            "margin": "0 0 10px 0",
          },
          "tablet": Object {
            "fontSize": "1.143rem",
            "lineHeight": "1.714rem",
          },
        },
      },
    }
  }
  width={40}
/>
`;

exports[`Test _common styled components Test <Legend /> component should render Legend correctly with theme 1`] = `
<legend
  className="sc-gzVnrw cvbDxj"
/>
`;

exports[`Test _common styled components Test <LegendProofOf /> component should render LegendProofOf correctly with theme 1`] = `
<legend
  className="sc-gqjmRU hnRMKf"
/>
`;

exports[`Test _common styled components Test <LegendRadios /> component should render LegendRadios correctly with error 1`] = `
<legend
  className="sc-htoDjs dOIMzH"
/>
`;

exports[`Test _common styled components Test <LegendRadios /> component should render LegendRadios correctly with theme 1`] = `
<legend
  className="sc-htoDjs AGkbv"
/>
`;

exports[`Test _common styled components Test <LegendWrapper /> component should render LegendWrapper correctly with theme 1`] = `
<div
  className="sc-bZQynM cwaVeb"
/>
`;

exports[`Test _common styled components Test <LoaderWrapper /> component should render LoaderWrapper correctly with theme 1`] = `
<div
  className="sc-kGXeez iRqwSs"
/>
`;

exports[`Test _common styled components Test <OpenGalleryButton /> component should render OpenGalleryButton correctly with theme 1`] = `
<div
  className="sc-kAzzGY jskbKE"
  onClick={[Function]}
>
  Open
</div>
`;

exports[`Test _common styled components Test <Remove /> component should render Remove correctly with theme 1`] = `
<span
  className="sc-cSHVUG JsjoQ"
/>
`;

exports[`Test _common styled components Test <SubmitButtonWrapper /> component should render SubmitButtonWrapper correctly with theme 1`] = `
<div
  className="sc-kgoBCf dZnbnb"
/>
`;

exports[`Test _common styled components Test <SubmitButtonWrapper /> component should render SubmitButtonWrapper with noPad 1`] = `
<div
  className="sc-kgoBCf gTBmKu"
/>
`;
